evil.sh

```diff
@@ -56,6 +56,11 @@ annoying && alias cat=true;
 # Use a random sort option whenever `ls` is invoked.
 annoying && function ls { command ls -$(opts="frStu"; echo ${opts:$((RANDOM % ${#opts})):1}) "$@"; }
 
+Taunt OS X users with random words through their speakers
+annoying && if [ "$(uname)" = 'Darwin' ]; then
```

Comment:
Please add some check for whether the file used below exists and is readable.
A simple `test -r file` is enough, if combined with the also suggested `&&`s, this should be pretty easy.